stdlib/public/core/VarArgs.swift

```diff
@@ -371,7 +384,7 @@ final internal class _VaListBuilder {
     internal var gp_offset = CUnsignedInt(0)
     @_versioned // FIXME(sil-serialize-all)
     internal var fp_offset =
-      CUnsignedInt(_x86_64CountGPRegisters * MemoryLayout<Int>.stride)
+      CUnsignedInt(_countGPRegisters * MemoryLayout<Int>.stride)
```

Comment:
Should we use `MemoryLayout<_VAUInt>.stride` here?

Reply:
@compnerd 
Here "MemoryLayout<Int>.stride", "Int" is 64 bits by default on 64 platform.  By the way, both `fp_offset` and `gp_offset`  are not used currently. It is better to keep as the original author intend.